tests/locale/keymap_or_locale.pm

```diff
@@ -0,0 +1,98 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Keyboard layout test in console and gdm after boot
+# Maintainer: Martin Loviska <mloviska@suse.com>
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+use utils;
+
+sub verify_default_keymap_textmode {
+    my ($test_string, $tag, $tty, $console) = @_;
+
+    if (defined($tty)) {
```

Comment:
you can use: `send_key($tty) if defined $tty` (more readable and of common use in ther repo).